internal/repos/importer.go

```diff
@@ -204,3 +215,54 @@ func (i *importer) handleDeletedRepos(ctx context.Context, tx pgx.Tx, repos []*g
 
 	return err
 }
+
+func (i *importer) handleDefaultSyncs(ctx context.Context, tx pgx.Tx, defaultSyncs []string, impID uuid.UUID) error {
+	var err error
+	var repoIDs []uuid.UUID
+
+	i.logger.Info().Msgf("starting to insert default syncs for import %s", impID)
+
+	if repoIDs, err = i.db.WithTx(tx).GetAllReposId(ctx, impID); err != nil {
+		i.logger.Err(err).Msgf("failed to retrieve repoids for import %s", impID)
+		return err
+	}
+
+	for _, repoID := range repoIDs {
+		if err = i.insertSyncInQueue(ctx, tx, defaultSyncs, repoID); err != nil {
+			i.logger.Err(err)
+			return err
+		}
+
+	}
```

Comment:
IMO we should remove this functionality from here, if I'm understanding this correctly. It looks like you're also enqueuing the newly added `repo_sync`s to run during the import? To me, that should be a separate process, we'll let the scheduler pick those up and enqueue with everything else. I think the code here should just _enable_ the default sync types for a repo, but not trigger it to run